### PR TITLE
Display multiple notifications

### LINKED ISSUE #200: Display multiple notifications
Currently, we can only display 1 notification at the same time, which means if there will be a need of displaying 2 or more separate alerts we would have to compute them into 1. 
We should change that to be able to display a couple of alerts at the same time.

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

fixes #200

### Screenshots

Screenshot 2019-05-09 at 16 51 14

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -69,54 +75,61 @@ const startApp = async () => {
...
+                  <UserProviderWithTokenHandler
+                    apolloClient={apolloClient}
+                    onUserLogin={() =>
+                      show(OverlayType.message, null, {
+                        title: "You are logged in"
+                      })
+                    }
+                    onUserLogout={() =>
+                      show(OverlayType.message, null, {
+                        title: "You are logged out"
```
Comment (NyanKiyoshi) [inline src/index.tsx]:
```suggestion
                        title: "You are now logged out"
```

File: src/index.tsx [hunk 2]
```diff
@@ -69,54 +75,61 @@ const startApp = async () => {
...
+        <AlertProvider template={NotificationTemplate} {...notificationOptions}>
+          <ShopProvider>
+            <OverlayProvider>
+              <OverlayContext.Consumer>
+                {({ show }) => (
+                  <UserProviderWithTokenHandler
+                    apolloClient={apolloClient}
+                    onUserLogin={() =>
+                      show(OverlayType.message, null, {
+                        title: "You are logged in"
```
Comment (NyanKiyoshi) [inline src/index.tsx]:
```suggestion
                        title: "You are now logged in"
```

File: src/checkout/views/Review/View.tsx [hunk 3]
```diff
@@ -41,15 +36,26 @@ const completeCheckout = (
...
-      title: "Your order was placed"
-    });
+    alert.show(
+      { title: "Your order was placed" },
+      {
+        type: "success"
+      }
+    );
+    alert.show(
+      { title: "Your order was placed again" },
```
Comment (NyanKiyoshi) [inline src/checkout/views/Review/View.tsx]: I wonder if we could come up with a better sentence than this one? It is sounding really weird. But I have no good suggestion in mind.

Comment (AlicjaSzu) [reply src/checkout/views/Review/View.tsx]: "Your order was placed again"  will be removed, it was added only to show how it looks with multiple notifications

File: src/checkout/views/Review/View.tsx [hunk 4]
```diff
@@ -60,92 +60,90 @@ const View: React.FC<RouteComponentProps<{ token?: string }>> = ({
...
+
+  return (
+    <>
+      {stepCheck}
+      <div className="checkout-review">
+        <Link
+          to={generatePath(paymentUrl, { token })}
+          className="checkout-review__back"
+        >
+          Go back to Previous Step
```
Comment (NyanKiyoshi) [inline src/checkout/views/Review/View.tsx]:
```suggestion
          Go back to the previous step
```

**Comment by piotrgrundas** [comment]
Please check which stuff is resolved from the comments.

File: src/checkout/views/Review/View.tsx (hunk 4 above)
Comment (AlicjaSzu) [reply src/checkout/views/Review/View.tsx]: updated

**Pull request merged**